src/screens/SettingsScreen.js

```diff
@@ -42,24 +45,22 @@ class SettingsScreen extends InjectedComponent<InjectedProps> {
     return (
       <Container>
         <Header title="Settings" />
-        {organizations.length < 1 ? null : (
-          <PickerField
-            key="organization"
-            label="Selected Organization"
-            name="organization"
-            onChange={onOrganizationChange}
-            placeholder="None"
-            value={selectedOrganizationID}
-          >
-            {organizations.map((organization: Organization): React.Node => (
-              <PickerField.Item
-                key={organization.id}
-                label={`${organization.id} - ${organization.name}`}
-                value={organization.id}
-              />
-            ))}
-          </PickerField>
-        )}
+        <LoaderPickerField
+          enabled={this._hasOrganizations}
+          itemsLoader={OrganizationStore.getMany()}
+          key="organization"
+          label="Selected Organization"
+          name="organization"
+          onChange={onOrganizationChange}
+          placeholder="None"
+          value={selectedOrganizationID}
+        >
+          {(items: Array<Organization>): Array<React.Node> =>
```

Comment:
I thought we were moving away from passing a function as `children` to components? You could make `LoaderPickerField` take in a component that expects `items` as a prop to render this.